en/getting_started/ros.md

```diff
@@ -0,0 +1,12 @@
+# Generate Source Files for ROS
+
+If you want to add MAVlink messages or dialects while working with ROS, you need to:
+
+- follow the [instructions](https://github.com/mavlink/mavros/blob/master/mavros/README.md#source-installation) to install the mavlink-gbp-release which is the MAVlink library released for ROS.
```

Comment:
It's preferable if you enumerations.

Reply:
`[MAVROS source install instructions](https://github.com/mavlink/mavros/blob/master/mavros/README.md#source-installation)`